### DISCUSSION TITLE
ICSx5 lost calendar permission

### DISCUSSION THREAD

**Discussion by septem9er** [bitfireAT/icsx5#208] [General]
Yesterday I noticed, that a calendar wasn't synced so I opened ICSx5 which then notified me that it is missing the calendar permission. It worked flawlessly for months, it definitely had the permission before.

Maybe because of the "Remove permission if App isn't used" feature? If so, maybe ICSx5 should inform the users that they may want to disable this for the app.

I am on Android 14, with ICSx5 Version 2.1.1 (72) from F-Droid.

Not a very helpful bug report, because I cannot tell you how to reproduce it. But I at least wanted to bring it to your attention.

**Comment by ArnyminerZ** [comment]
Yeah, there's really not any way to reproduce it, the permission is revoked by Android automatically. But yes, I agree that we should show a warning just like in DAVx5. What do you think @rfc2822? Maybe we can just show a new card, like the calendar permission

**Reply by rfc2822** [reply]
And/or a notification when sync is performed and doesn't have permissions (anymore)?

**Reply by ArnyminerZ** [reply]
Maybe. Now that I think about it, I don't know if it's that much necessary to overcomplicate it. We already have the check for missing permissions. Maybe we can just show a dismissable warning or something like that, and in any case, if the permission is revoked, the missing permission notification will appear when the sync process is performed.

**Reply by septem9er** [reply]
I didn't see a notification. And if the issue is that android removes the permission because of the "pause app activity if unused" feature, then that makes sense because it says it will stop notifications too.

**Reply by ArnyminerZ** [reply]
> then that makes sense because it says it will stop notifications too

Absolutely, didn't think about that one. Then yeah, we should add the warning. I'll create an issue
